Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -238,6 +233,23 @@ public struct Configuration {
     }
 }
 
+func errorMessage(for error: Error, hasCustomConfigurationFiles: Bool) -> String? {
```

Comment:
It would be good to have this method marked as `private`, but that's an optional change.

Reply:
Also, maybe there is a better name of this function? In principle, it decides which error is an actual error and then only maps the actual errors to strings. Maybe `flatMapToErrorMessage(error:hasCustomConfigurationFiles:)`. That's also not that great but at least includes a hint to a potential `nil` return value.